command_stream.py

```diff
@@ -1,34 +1,48 @@
-from serverless_sdk import tag_event
-from clients import sqs
-from utils.kinesis import get_payloads_from_kinesis_event
+import logging
+from utils.kinesis import get_payload_from_kinesis_record
 
-
-COMMAND_TYPES = {"INBOUND_SMS"}
+from dialog.dialog import (
+    process_command,
+    ProcessSMSMessage,
+    StartDrill,
+    TriggerReminder,
+)
+from drills.drills import drill_from_dict
 
 
 def handle_command(raw_event, context):
-    tag_event("command_stream", "handle_command", raw_event)
-    events = get_payloads_from_kinesis_event(raw_event)
-    tag_event("command_stream", "commands", events)
-
-    responses = [get_response(event) for event in events]
 
-    # temporary for now. eventually this should write to the dialog log
-    sqs.publish_outbound_sms_messages(responses)
+    for record in raw_event["Records"]:
+        event = get_payload_from_kinesis_record(record)
+        command_type = event["type"]
+        payload = event["payload"]
+        sequence_number = record["kinesis"]["sequenceNumber"]
+
+        if command_type == "INBOUND_SMS":
+            process_command(
+                ProcessSMSMessage(
+                    phone_number=payload["From"], content=payload["Body"]
+                ),
+                sequence_number,
+            )
+        elif command_type == "START_DRILL":
+            process_command(
+                StartDrill(
+                    phone_number=payload["phone_number"], drill=drill_from_dict(payload["drill"])
+                ),
+                sequence_number,
+            )
+        elif command_type == "TRIGGER_REMINDER":
+            process_command(
+                TriggerReminder(
+                    phone_number=payload["phone_number"],
+                    drill_id=payload["drill_id"],
```

Comment:
Instead of `drill_id` I think we need `drill_instance_id` here. Let me know if you'd like me to fix as there's some cleanup of the TriggerReminder command and downstream events that may be needed.